### ISSUE TITLE
Cannot build 2.6.0 Debian package from source (release tarball)

### ISSUE THREAD

**Issue by steffen-AEI** [git-lfs/git-lfs#3362]
In the past (at least up to 2.5.1, I didn't check 2.5.2), I was able to pull the v${version}.tar.gz tarball from Releases, unpack/rename it and run dpkg-source -b to create a proper Debian source package, which I injected into a pbuilder and obtained a binary package. (There are more architectures than amd64, arm64 and i386. There are people using Raspbian...)
With 2.6.0, the build fails:
```
[...]
dh_auto_build
        go install -v -p 1 github.com/git-lfs/git-lfs github.com/git-lfs/git-lfs/commands github.com/git-lfs/git-lfs/config github.com/git-lfs/git-lfs/creds github.com/git-lfs/git-lfs/docs/man github.com/git-lfs/git-lfs/errors github.com/git-lfs/git-lfs/filepathfilter github.com/git-lfs/git-lfs/fs github.com/git-lfs/git-lfs/git github.com/git-lfs/git-lfs/git/gitattr github.com/git-lfs/git-lfs/git/githistory github.com/git-lfs/git-lfs/lfs github.com/git-lfs/git-lfs/lfsapi github.com/git-lfs/git-lfs/lfshttp github.com/git-lfs/git-lfs/locking github.com/git-lfs/git-lfs/script github.com/git-lfs/git-lfs/subprocess github.com/git-lfs/git-lfs/t/cmd github.com/git-lfs/git-lfs/t/cmd/util github.com/git-lfs/git-lfs/tasklog github.com/git-lfs/git-lfs/tools github.com/git-lfs/git-lfs/tools/humanize github.com/git-lfs/git-lfs/tools/kv github.com/git-lfs/git-lfs/tq github.com/git-lfs/git-lfs/vendor/github.com/ThomsonReutersEikon/go-ntlm/ntlm github.com/git-lfs/git-lfs/vendor/github.com/ThomsonReutersEikon/go-ntlm/ntlm/md4 github.com/git-lfs/git-lfs/vendor/github.com/git-lfs/gitobj github.com/git-lfs/git-lfs/vendor/github.com/git-lfs/gitobj/errors github.com/git-lfs/git-lfs/vendor/github.com/git-lfs/gitobj/pack github.com/git-lfs/git-lfs/vendor/github.com/git-lfs/gitobj/storage github.com/git-lfs/git-lfs/vendor/github.com/git-lfs/go-netrc/netrc github.com/git-lfs/git-lfs/vendor/github.com/git-lfs/wildmatch github.com/git-lfs/git-lfs/vendor/github.com/inconshreveable/mousetrap github.com/git-lfs/git-lfs/vendor/github.com/rubyist/tracerx github.com/git-lfs/git-lfs/vendor/github.com/spf13/pflag github.com/git-lfs/git-lfs/vendor/golang.org/x/sys/unix
src/github.com/git-lfs/git-lfs/vendor/github.com/stretchr/testify/assert/assertions.go:19:2: cannot find package "github.com/davecgh/go-spew/spew" in any of:
        /build/git-lfs-2.6.0/obj-x86_64-linux-gnu/src/github.com/git-lfs/git-lfs/vendor/github.com/davecgh/go-spew/spew (vendor tree)
        /usr/lib/go-1.8/src/github.com/davecgh/go-spew/spew (from $GOROOT)
        /build/git-lfs-2.6.0/obj-x86_64-linux-gnu/src/github.com/davecgh/go-spew/spew (from $GOPATH)
src/github.com/git-lfs/git-lfs/vendor/github.com/stretchr/testify/assert/assertions.go:20:2: cannot find package "github.com/pmezard/go-difflib/difflib" in any of:
        /build/git-lfs-2.6.0/obj-x86_64-linux-gnu/src/github.com/git-lfs/git-lfs/vendor/github.com/pmezard/go-difflib/difflib (vendor tree)
        /usr/lib/go-1.8/src/github.com/pmezard/go-difflib/difflib (from $GOROOT)
        /build/git-lfs-2.6.0/obj-x86_64-linux-gnu/src/github.com/pmezard/go-difflib/difflib (from $GOPATH)
dh_auto_build: go install -v -p 1 github.com/git-lfs/git-lfs github.com/git-lfs/git-lfs/commands github.com/git-lfs/git-lfs/config github.com/git-lfs/git-lfs/creds github.com/git-lfs/git-lfs/docs/man github.com/git-lfs/git-lfs/errors github.com/git-lfs/git-lfs/filepathfilter github.com/git-lfs/git-lfs/fs github.com/git-lfs/git-lfs/git github.com/git-lfs/git-lfs/git/gitattr github.com/git-lfs/git-lfs/git/githistory github.com/git-lfs/git-lfs/lfs github.com/git-lfs/git-lfs/lfsapi github.com/git-lfs/git-lfs/lfshttp github.com/git-lfs/git-lfs/locking github.com/git-lfs/git-lfs/script github.com/git-lfs/git-lfs/subprocess github.com/git-lfs/git-lfs/t/cmd github.com/git-lfs/git-lfs/t/cmd/util github.com/git-lfs/git-lfs/tasklog github.com/git-lfs/git-lfs/tools github.com/git-lfs/git-lfs/tools/humanize github.com/git-lfs/git-lfs/tools/kv github.com/git-lfs/git-lfs/tq github.com/git-lfs/git-lfs/vendor/github.com/ThomsonReutersEikon/go-ntlm/ntlm github.com/git-lfs/git-lfs/vendor/github.com/ThomsonReutersEikon/go-ntlm/ntlm/md4 github.com/git-lfs/git-lfs/vendor/github.com/git-lfs/gitobj github.com/git-lfs/git-lfs/vendor/github.com/git-lfs/gitobj/errors github.com/git-l
fs/git-lfs/vendor/github.com/git-lfs/gitobj/pack github.com/git-lfs/git-lfs/vendor/github.com/git-lfs/gitobj/storage github.com/git-lfs/git-lfs/vendor/github.com/git-lfs/go-netrc/netrc github.com/git-lfs/git-lfs/vendor/github.com/git-lfs/wildmatch github.com/git-lfs/git-lfs/vendor/github.com/inconshreveable/mousetrap github.com/git-lfs/git-lfs/vendor/github.com/rubyist/tracerx github.com/git-lfs/git-lfs/vendor/github.com/spf13/pflag github.com/git-lfs/git-lfs/vendor/golang.org/x/sys/unix returned exit code 1
debian/rules:35: recipe for target 'override_dh_auto_build' failed
make[1]: *** [override_dh_auto_build] Error 1
[...]                                                                                                                                                                                              
```

Apparently, the release tarball is no longer self-consistent, or some files haven't been modified to match the recent changes.
Any suggestions how to overcome this?

**Comment by bk2204** [comment]
Thanks for reaching out. I think this will be fixed with #3356 and we'll be doing a 2.6.1 soon to address this issue.

In the mean time, you could apply the patch from that pull request as a patch in the Debian package, which should let things build.

**Comment by steffen-AEI** [comment]
Um... I had checked the issues, but forgot to have a look into the merge requests.
I had already, rather half-heartedly and not really knowing what I was doing, copied the missing parts from the 2.5.1 vendor tree, but obviously there was more to the job.

I vaguely remember that git-lfs required network access to build (a while ago), is this still true, and wouldn't it be possible to fetch missing vendor stuff during builds? (Pbuilders can be tricked into allowing network access although that's non-default behaviour.)

In the meantime, pbuilders for amd64 Jessie and Stretch finished successfully,. More will follow tomorrow. Thanks for the quick workaround!
